src/GeneralStateTestsFiller/stExample/eip1559Filler.yml

```diff
@@ -0,0 +1,84 @@
+eip1559:
+  _info:
+    comment: Ori Pomerantz   qbzzt1@gmail.com
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentNumber: 1
+    currentTimestamp: 1000
+    currentGasLimit: 0x1000000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+    # New in London because of EIP1559
+    currentBaseFee: 1000
+
+  pre:
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: '1000000000000000000'
+      code: |
+          :yul {
+             sstore(0,gasprice())
+             // basefee() is not supported yet by Yul, so we'll call a 
+             // machine language contract for it
+             pop(call(gas(), 0xBA5E0FEE, 0, 0, 0, 0, 0))
+          }
+      nonce: 1
+      storage: {}
+
+
+    00000000000000000000000000000000BA5E0FEE:
+      balance: '1000000000000000000'
+      #  BASEFEE    0x48
+      #  PUSH1 0    0x6000
+      #  SSTORE     0x55
+      #  STOP       0x00
+      code: :raw 0x4860005500
```

Comment:
basefee LLLC opcode?

Reply:
Oops, sorry - I forgot about it. Fixed now.